python/shark_turbine/dynamo/passes.py

```diff
@@ -42,6 +42,9 @@
     torch.ops.aten._adaptive_avg_pool2d,
     torch.ops.aten._prelu_kernel,
     torch.ops.aten.full,
+    torch.ops.aten._log_softmax,
```

Comment:
Note for the future: it isn't clear to me anymore what the default and cpu pass distinctions are. I think we likely just want all of these all of the time.